src/json.json

```
@@ -0,0 +1,54 @@
+{
+    "id": "6283598291e6b519379c26b4",
+    "propertyId": "2033",
+    "jurisdiction": null,
+    "year": 2022,
+    "month": 5,
+    "startDate": "2022-04-30T14:00:00.000Z",
+    "endDate": "2022-05-31T13:59:59.999Z",
+    "note": "",
```

```suggestion
    "note": null,
```

lets change like this